### PR TITLE
Map two inner classes

### PR DESCRIPTION
**Pull request by liach**
Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

File: mappings/net/minecraft/item/CompassItem.mapping [hunk 1]
```diff
@@ -15,3 +15,14 @@ CLASS net/minecraft/class_1759 net/minecraft/item/CompassItem
...
 	METHOD method_26365 hasLodestone (Lnet/minecraft/class_1799;)Z
 		ARG 0 stack
+	CLASS 1
+		FIELD field_23978 targeted Lnet/minecraft/class_1759$class_5171;
+		FIELD field_23979 targetless Lnet/minecraft/class_1759$class_5171;
+	CLASS class_5171 AngleRandomizer
+		FIELD field_23980 value D
+		FIELD field_23981 speed D
+		FIELD field_23982 lastUpdateTime J
+		METHOD method_27188 get ()D
```
Comment (Runemoro) [inline mappings/net/minecraft/item/CompassItem.mapping]: `targeted.get()` looks weird. Shouldn't be either `targetedAngleRandomizer.get()` or `targeted.getAngle()`?

File: mappings/net/minecraft/block/entity/JigsawBlockEntity.mapping [hunk 2]
```diff
@@ -25,3 +25,14 @@ CLASS net/minecraft/class_3751 net/minecraft/block/entity/JigsawBlockEntity
 			ARG 3 name
 		METHOD method_26401 byName (Ljava/lang/String;)Ljava/util/Optional;
 			ARG 0 name
+	CLASS class_5173 RuntimeStructurePiece
```
Comment (Juuxel) [inline mappings/net/minecraft/block/entity/JigsawBlockEntity.mapping:28]: Runtime?

Comment (liach) [reply mappings/net/minecraft/block/entity/JigsawBlockEntity.mapping:28]: Hmm, `JigsawBlockEntity.Runtime` doesn't sound like a... structure piece

Comment (Juuxel) [reply mappings/net/minecraft/block/entity/JigsawBlockEntity.mapping:28]: I meant like what does `Runtime` mean in this case?

Comment (liach) [reply mappings/net/minecraft/block/entity/JigsawBlockEntity.mapping:28]: a runtime structure piece. name from structure piece type

File: mappings/net/minecraft/item/CompassItem.mapping (hunk 1 above)
Comment (Juuxel) [reply mappings/net/minecraft/item/CompassItem.mapping]: `getAngle` sounds good.

**Review by YanisBft — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Review by Runemoro — APPROVED** [review]

**Pull request merged**